.github/actions/javascript/markPullRequestsAsDeployed/markPullRequestsAsDeployed.ts

```diff
@@ -120,7 +109,30 @@ async function run() {
                 repo: CONST.APP_REPO,
                 pull_number: prNumber,
             });
-            const deployer = isCP ? commit.committer.name : pr.merged_by?.login;
+
+            // Check for the CP Staging label on the issue to see if it was cherry-picked
+            const isCP = pr.labels.some(({name: labelName}) => labelName === 'CP Staging');
```

Comment:
Let's add a [const](https://github.com/Expensify/App/blob/87662420e7d199fc965a95f10efcf081f2f8be55/.github/libs/CONST.ts#L12-L17) for the label name?

Reply:
done